tests/vendir.yml

```diff
@@ -36,13 +36,29 @@ directories:
     includePaths:
       - examples/contour/*.yaml
     newRootPath: examples/contour
-  - path: testApps/dora
+
+- path: e2e/assets/node
+  contents:
+  - path: .
     git:
-      url: https://github.com/cloudfoundry/cf-acceptance-tests
+      url: https://github.com/cloudfoundry/cf-acceptance-tests.git
       depth: 1
       refSelection:
         semver:
           constraints: ">=10.0.0"
+    newRootPath: assets/node
     includePaths:
-      - assets/dora/**/*
+    - assets/node/**/*
+
+- path: e2e/assets/dora
+  contents:
+  - path: .
+    git:
+      url: https://github.com/cloudfoundry/cf-acceptance-tests.git
+      depth: 1
+      refSelection:
+        semver:
+          constraints: ">=10.0.0"
     newRootPath: assets/dora
```

Comment:
`e2e/assets` contains a mix of non-vendored and vendored apps now, right? I feel like it might get confusing if someone makes a change to `node` or `dora` (e.g. if we add endpoints that look at service binding files) and doesn't know these are actually managed through `vendir`.

Reply:
I liked that the original path made it was clear it was vendored... I am wondering if we could put this in a subdir e.g. `e2e/assets/vendored/dora` or some other convention involving the directory name or something. I just think the `vendir.yml` is pretty far removed from where these files actually live and it could be confusing.

Reply:
`e2e/assets/vendored/dora` LGTM, thanks!